client/components/filter-rank.jsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import Select from 'react-select';
+export default class FilterRank extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      selectedOption: null
+    };
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  handleChange = selectedOption => {
+    this.setState({ selectedOption });
+  };
+
+  render() {
+    const ranks = [
+      {
+        value: null,
+        label: (
+          <>
+            <span> Select Rank</span>
+          </>
+        )
+      }, {
+        value: 'Iron',
+        label: (
+        <>
+        <img className='select-rank-icon' src='/images/ranked-emblems/iron.png'></img>
+        <span> Iron</span>
+        </>
```

Comment:
I thought you had the ranks in your database? I recommend fetching them in `componentDidMount` and then using `map()` to make these elements.